test/spec/LintingSpec.js

```diff
@@ -547,6 +547,33 @@ describe('linting', function() {
 
         });
 
+
+        it('should position overlay on root element - task without BPMNDI', function(done) {
+
+          // given
+          const diagram = require('./no-bpmndi.bpmn');
+
+          // when
+          modeler.importXML(diagram).then(function() {
```

Comment:
Could be an `async` test case.

Reply:
Hmmm, can we sync regarding this?

`toggleLinting` takes a callback which is passed ot `eventBus.once`. And since `eventBus.once` is not promisified I think this does not work, am I wrong? (At least out of the box). 

What we could do: wrap around the eventBus.once and promisify it.

Reply:
My bad, I did not recognize the asyncronicity (toggleLinting) again.

We could make `toggleLinting` awaitable for the sake of cleaner tests fo course. But I#m fine the way it is.